include/irr/asset/normal_quantization.h

```diff
@@ -15,6 +15,28 @@ namespace irr
 {
 namespace asset
 {
+	struct QuantNormalHash
+	{
+		size_t operator()(const core::vectorSIMDf& vec) const noexcept
+		{
+			static constexpr uint32_t primeNumber1 = 73856093;
+			static constexpr uint32_t primeNumber2 = 19349663;
+			static constexpr uint32_t primeNumber3 = 83492791;
```

Comment:
if you're hashing in 64bits then you need 64bit long prime numbers (big big numbers)

Also make the hash 2D, cause you'll be storing projections